### PR TITLE
Integrate 'Check processor vulnerability mitigations' into openQA

### PR DESCRIPTION
**Pull request by Amrysliu**
Related: https://progress.opensuse.org/issues/111809

Verify run:
https://openqa.suse.de/tests/8863389# (64bit)
https://openqa.suse.de/tests/8863390# (aarch64)

### CODE REVIEW

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by wangqianzou — APPROVED** [review]
LGTM

File: tests/security/atsec/check_processor_vulnerability_mitigations.pm [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    foreach my $item (split(/\s/, $summary)) {
+        if ($item !~ /(^CVE-.*):(\w+)/) {
+            next;
+        }
+        my $name = $1;
+        my $result = $2;
+        if ($result ne 'OK') {
+            record_info($name, "Check failed: $item, please see $log_file for more details", result => 'fail');
+
+            # CVE-2018-3639 will be KO on aarch64 OSD worker.
```
Comment (jouyingbin) [inline tests/security/atsec/check_processor_vulnerability_mitigations.pm:49]: @Amrysliu perhaps it will be OK instead of KO, am I right? :-)

Comment (jouyingbin) [reply tests/security/atsec/check_processor_vulnerability_mitigations.pm:49]: Sorry , looks like I am wrong

**Review by jouyingbin — APPROVED** [review]
LGTM

**Pull request merged**